### DISCUSSION TITLE
onChange doesn't working with MUI-pickers

### DISCUSSION THREAD

**Discussion by Oxyps** [react-hook-form/react-hook-form#2234] [Q&A] [answered]
Im using react-hook-form v6 and mui-pickers v3-latest.
The code without Controller component woks just fine, but when I try control the component, the `onChange` prop doesnt even hit the console. There's something im missing in how to use the module, and dont no what is.
Here is a sandbox https://codesandbox.io/s/awesome-cherry-j3kt0?file=/src/App.js.
Appreciate if someone could help, 2 days stuck in that ç.ç

**Comment by bluebill1049** [comment, marked as answer]
hey @Oxyps sorry to hear that, you are using v6, you should take a look `render` method, `onchange` is for v5. 

https://react-hook-form.com/api/#Controller

https://codesandbox.io/s/react-hook-form-v6-controller-qsd8r

**Reply by Oxyps** [reply]
Cool, now it works.
You're awesome!, thanks for the light, Bill.
In case someone else needs, [here's](https://codesandbox.io/s/heuristic-https-z22ic?file=/src/App.js) the code.

**Reply by albertolotz** [reply]
Your code help me a lot! tks
